portal/templates/portal_wrapper.html

```diff
@@ -92,10 +56,9 @@
 <noscript><p><img src="//piwik.cirg.washington.edu/piwik.php?idsite={{ config.PIWIK_SITEID }}" style="border:0;" alt="" /></p></noscript>
 {% endif %}
 {% if not enable_links %}
-<style>
-    #tnthNavXs .tnth-nav-btn{ display: none; }
-</style>
+<style>#tnthNavXs .tnth-nav-btn{ display: none; }</style>
 {% endif %}
+{% if config.SYSTEM_TYPE and config.SYSTEM_TYPE != "Production" and config.SYSTEM_TYPE != "production" %}<div class='watermark'>TRUE<sup>NTH</sup> - {{config.SYSTEM_TYPE}} version - Not for study or clinical use</div>{% endif %}
```

Comment:
I might change this a little- `SYSTEM_TYPE` will always be a defined config value since it's in [`config.py`](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/config.py#L105) (no need to check `if config.SYSTEM_TYPE`) and this check might miss someone who configures `SYSTEM_TYPE` to be "PRODUCTION". 

Something a bit more robust (and terse):
```
{% if config["SYSTEM_TYPE"].lower() == "production" %}
```